### DISCUSSION TITLE
Breed axolots without a bucket

### DISCUSSION THREAD

**Discussion by MrRaposinha** [PurpurMC/Purpur#731] [__Dismissed__] [answered]
I ended up finding this plugin and thought it might be a good option to add to Purpur! :3

https://www.spigotmc.org/resources/bucketlessaxolotlbreeding.97077/

**Comment by BillyGalbreath** [comment, marked as answer]
You can do this with a simple data pack to change the axolotl temp item tag. (tempt is also used as the breeding item for breedable mobs)

Vanilla tag `axolotl_tempt_items.json`

```json
{
  "replace": false,
  "values": [
    "minecraft:tropical_fish_bucket"
  ]
}
```

Just change the item to `minecraft:tropical_fish` and you're all set. If you want it to _replace_ the original bucket item to the tag instead of appending it, change `replace` to true.

Here's a data pack I made in 10 seconds that appends the item ;)

[BreedAxolotlNoBucket.zip](https://github.com/PurpurMC/Purpur/files/7796649/BreedAxolotlNoBucket.zip)
